javascript/javascript1/week3/travel-time-calculation.js

```diff
@@ -0,0 +1,18 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function timeCalculate(info) {
```

Comment:
I'll give you some rules we follow at work for function names.
- almost the function name starts with a verb
- when the function returns something, then it could be a good idea to give the function name `get` + `the thing returns`
So your function could have a name `getTime` instead of `timeCalculate`